packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -92,6 +92,7 @@ const AdvertiserPage = ({ navigate, selected_advert, showVerification }) => {
         total_completion_rate,
         total_orders_count,
     } = stats;
+    const avg_release_time = release_time_avg > 60 ? (release_time_avg / 60).toFixed(2) : '< 1';
```

Comment:
```suggestion
    const release_time_avg_text = release_time_avg > 60 ? (release_time_avg / 60).toFixed(2) : '< 1';
```

Reply:
We should round the average release time to the nearest minute. e.g. `2.7` minutes become `3`, `2.3` minutes becomes `2`. For anything below `1` minute, `< 1 min` is correct.

Reply:
@salar-fs Thank you for the suggestion! However, I don't think `text` would work because it's a placeholder in `localize` which would contain the entire text. I like @aaron-binary's suggestion to use `avg_release_time_in_minutes`, it sounds the most informative. 😄

Reply:
@aaron-binary This is how it was in the _old_ design, since it's been updated, I'll update the code as well, thanks! :)